enterprise-suite/scripts/lbc.py

```diff
@@ -92,6 +92,12 @@ def run(cmd, timeout=None, stdin=None, show_stderr=True):
         if len(stderr) > 0 and show_stderr:
             printerr(stderr)
         returncode = proc.returncode
+    except OSError, o:
+        stdout=o.strerror
+        returncode=o.errno
+    except Exception, e:
+        stdout=str(e)
+        returncode=1
```

Comment:
I don't think we should catch the exception here every time. For example if we are trying to run the `helm` command, then it should fail with an exception.

Instead in is_running_minishift, we should catch OSError in case it doesn't exist. WDYT?

Reply:
We have a return in the finally block, so the exceptions are never raised, 
I choose this way, to catch the exception and return a non-zero error code, and leave it for the caller to handle it.. 
Note: Lot us didn't have minishift command installed, and there was no exception (errno 2) for that.